### PR TITLE
Updating wolox eslint version

### PR DESCRIPTION
**Pull request by raparicio6**
I am also adding `prettier-eslint` package since is required in Wolox linter https://github.com/Wolox/eslint-config-wolox/blob/master/package.json#L17 but it wasnt here

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+
+## About
+
+This project is maintained by [Wolox](https://github.com/wolox) and it was written by [Wolox](http://www.wolox.com.ar).
+
+![Wolox](https://raw.githubusercontent.com/Wolox/press-kit/master/logos/logo_banner.png)
+
+## License
+
+**eslint-config-wolox-node** is available under the MIT [license](LICENSE.md).
```
Comment (pabloferro) [inline README.md]:
```suggestion
**eslint-config-wolox-react** is available under the MIT [license](LICENSE.md).
```

**Review by pabloferro — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (raparicio6) [reply README.md]: lol

**Comment by raparicio6** [comment]
@pabloferro @damfinkel don't forget me :cry:

**Review by damfinkel — APPROVED** [review]

**Review by pabloferro — APPROVED** [review]

**Review by damfinkel — CHANGES_REQUESTED** [review]
prettier-eslint is [deprecated](https://github.com/prettier/prettier-vscode/issues/958) , I'll close this PR

**Pull request closed without merging**